cpp/ps0008/p0008.cpp

```diff
@@ -1,31 +1,46 @@
 #include <iostream>
-#include <limits>
 #include <cctype>
 #include <string>
 using namespace std;
+
+// Decrypting process 3/3
 void decrypt(string msg, int key)
 {
     for (char& c : msg) {
-        if (isupper(c)) {    
+        if (isupper(c)) {
             c = (c - 'A' - key + 26) % 26 + 'A';
         } else if (islower(c)) {
             c = (c - 'a' - key + 26) % 26 + 'a';
         }
     }
     cout << "Decrypted message: " << msg << endl;
 }

```

Comment:
- [x] defined function takes two parameters (ciphertext and key)for decryption process
- [x] considered the case of the letter in the message while decryption
- [x] skip numbers and other special characters while decryption
- [x] considered that the key may be positive or negative value
- [x] returned/printed the actual message (message before encryption)